backend/FwLite/MiniLcm.Tests/CreateEntryTestsBase.cs

```diff
@@ -28,7 +28,9 @@ public async Task CanCreateEntry_AutoFaker()
             .For(e => e.Components).Exclude(e => e.Id)
             .For(e => e.ComplexForms).Exclude(e => e.Id)
             //predefined is always true in fwdata bridge, so we need to exclude it for now
-            .For(e => e.Senses).For(s => s.SemanticDomains).Exclude(s => s.Predefined));
+            .For(e => e.Senses).For(s => s.SemanticDomains).Exclude(s => s.Predefined)
+            .For(e => e.Senses).Exclude(s => s.PartOfSpeech!.Predefined)
```

Comment:
once #1344 is merged in both of these exclusions can be removed as Predefined will be calculated correctly.

Note I used `!` here to disable the error about PartOfSpeech being null, because this code is an expression (which preventing you from using `?`) it's not actually executed, it's just parsed to figure out which property you are referring to, meaning you can use `!` safely.

Reply:
Have now rebased on top of `develop` after #1344's merge, and pushed some new commits at the same time, so I eliminated even this exclusion. All tests are passing now, at least locally. (GHA tests still running as I'm writing this).